sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -172,66 +274,22 @@ for (int i = 0; i < replaceInstancesResult.Value.Length; i++)
 }
 ```
 
-### Delete instances
-
-To delete instances, pass in a list of time series Ids or names for the instances you want to delete.
-```C# Snippet:TimeSeriesInsightsSampleDeleteInstanceById
-var instancesToDelete = new List<TimeSeriesId>
-{
-    tsId,
-};
+## Time Series Insights Types
+Use [Types](https://github.com/Azure/azure-sdk-for-net/blob/82c7aff5ef867f7a3d77f5ac43a0b1c5576996e6/sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs#L42) in the `TimeSeriesInsightsClient` to create, retrieve, replace and delete Time Series types in your environment.
```

Comment:
same here update link with master branch code

Reply:
this is by design. Reason being is that if the TimeSeriesInsightsClient code changes, the line number '42' is no longer relatives. So a permlink will prevent that from happening.

Reply:
fixed to point to master